containers/docker-compose-testing.yml

```diff
@@ -27,6 +27,22 @@ services:
       - RAILS_ENV=${RAILS_ENV}
       - SECRET_KEY_BASE=${SECRET_KEY_BASE}
       - REDIS_URL=redis://redis:6379/0
+      - DANGER_GITHUB_API_TOKEN=${DANGER_GITHUB_API_TOKEN}
+      - TRAVIS_PULL_REQUEST=${TRAVIS_PULL_REQUEST}
+      - TRAVIS_REPO_SLUG=${TRAVIS_REPO_SLUG}
+      - CI=${CI}
+      - TRAVIS=${TRAVIS}
+      - CONTINUOUS_INTEGRATION=${CONTINUOUS_INTEGRATION}
+      - HAS_JOSH_K_SEAL_OF_APPROVAL=${HAS_JOSH_K_SEAL_OF_APPROVAL}
```

Comment:
Was this just for demonstration purposes, @icarito ?

Reply:
I don't remember off the top of my head why HAS_JOSH_K_SEAL_OF_APPROVAL is needed but it is no invention of mine and it seems some libs check for it!

Reply:
https://docs.travis-ci.com/user/environment-variables/